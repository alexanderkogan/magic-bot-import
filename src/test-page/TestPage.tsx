import * as React from 'react'

import Title from '../pattern-library/Title'
import Heading from '../pattern-library/Heading'
import SecondTestContent from './MtgCardsContent'

export default class TestPage extends React.Component {
    public render(): JSX.Element {
        return (
            <div>
                <Title>
                    <Heading>Fetch MTG cards</Heading>
                </Title>
                <SecondTestContent />
            </div>
        )
    }
}
