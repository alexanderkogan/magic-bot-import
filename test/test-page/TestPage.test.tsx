import * as React from 'react'

import TestPage from '../../src/test-page/TestPage'
import { shallow } from '../test-utils'

describe('TestPage', () => {
    it('renders shallow', () => {
        const element = shallow(<TestPage />)
        expect(element).toMatchSnapshot()
    })
})
