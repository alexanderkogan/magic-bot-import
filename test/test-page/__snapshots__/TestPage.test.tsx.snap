// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TestPage renders shallow 1`] = `
<div>
  <Component>
    <Component>
      Fetch MTG cards
    </Component>
  </Component>
  <Connect(Component) />
</div>
`;
